packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -293,6 +359,8 @@ export function isNetwork(chainId: ChainId) {
     isArbitrumRinkeby,
     isArbitrumGoerli,
     isArbitrumSepolia,
+    // Orbit chains
+    isOrbitChain: !isEthereum && !isArbitrum,
```

Comment:
Chains unrelated to Arbitrum or Ethereum won't be part of the UI. So pretty much anything that's not Arbitrum or Ethereum is an Orbit chain

Reply:
I would still side with having it logically correct. Since in the future, for wrong network identification, we might use this function and this might result in incorrect results. `isValidNetwork = isEthereum || isArbitrum || isOrbitChain`